activerecord/test/models/face.rb

```diff
@@ -2,6 +2,7 @@
 
 class Face < ActiveRecord::Base
   belongs_to :man, inverse_of: :face
+  belongs_to :autosave_man, class_name: "Man", foreign_key: :autosave_man_id, inverse_of: :autosave_face
```

Comment:
```suggestion
  belongs_to :autosave_man, class_name: "Man", foreign_key: :man_id, inverse_of: :autosave_face
```

(This will be `human_id` after a rebase.)